CONTRIBUTING.md

```diff
@@ -217,6 +217,41 @@ Please remember to:
 * Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
 
 
+## Writing Sequential Notebooks
+
+Most notebooks are singular - all information and code needed for that example is stored in one notebook. However, there are a few cases in which an example may be split into multiple notebooks. These are called sequential notebooks, as the sequence of the example is split among multiple notebooks. An example you can look at is [this series of sequential notebooks that demonstrate how to build a music recommender](https://github.com/aws/amazon-sagemaker-examples/tree/main/end_to_end/music_recommendation).
+
+### When should Sequential Notebooks be used?
+
+You may want to consider using sequential notebooks to write your example if the following conditions apply:
+
+* Your example takes over two hours to execute
+* Your notebook is extremely lengthy to the point that it hinders the learning experience
```

Comment:
Try to clarify how to define "lengthy"... number of code cells? Number of sections? Otherwise this will be inconsistent across contributors' judgments.